torchvision/transforms/transforms.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, dtype: torch.dtype) -> None:
         self.dtype = dtype
 
     def __call__(self, image: torch.Tensor) -> torch.Tensor:
-        return F.convert_image_dtype(image, self.dtype)
+        return F_t.convert_image_dtype(image, self.dtype)
```

Comment:
Can we revert the changes in this file, given that we have now exposed `convert_image_dtype` to `functional.py`?
This way `transforms.py` only depends on `functional.py`